### PR TITLE
project manipulating images V.1

### PR DESCRIPTION
**Pull request by ryby6969**

### CODE REVIEW

**Comment by ryby6969** [comment]
I get no errors but when I run it on my computer it creates the withLogo folder but nothing goes in it. I also do not get the response in IDLE when I run it with the "Adding logo to" response.

**Comment by halfaro1** [comment]
Closing and reopening the pull request to force the automated check to run...

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -7,8 +7,35 @@
...
+    if not(filename.endswith('.png') or filename.endswith('.jpg'))\
+       or filename == LOGO_FILENAME:
+        continue #skip non-image files and the logo itself
+    im = Image.open(filename)
+    width, height = im.size
+
+#TODO: Check if file image needs to be resized.
+    if width > SQUARE_FIT_SIZE and height > SQUARE_FIT_SIZE:
 #TODO: Calculate the new width and height to resize to.
+    if width > height:
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @ryby6969 because this line is below another `if` statement, shouldn't it be indented?

**Comment by ryby6969** [comment]
Fixed that and it still fails.

File: resizeAndAddLogo.py [hunk 2]
```diff
@@ -7,8 +7,35 @@
 logoIm = Image.open( LOGO_FILENAME )
 logoWidth, logoHeight = logoIm.size
 
+os.makedirs('withLogo', exist_ok=True)
 #TODO: Loop over all files in the working directory
-#TODO: Check if file image needs to be
+
+for filename in os.listdir('.'):
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @ryby6969 check out my comment on Sean's pull request, you're having the same issue: https://github.com/valencia-cop1000-201620/project-manipulating-images/pull/2#discussion_r59284812

**Comment by ryby6969** [comment]
I tried with just changing it to 'originals' and tried with the join function and neither worked.

**Comment by mmarchetti2** [comment]
Hey, quick thing I thought I'd mention, in line 40, there's 'imsave' instead of 'im.save'. I'm pretty sure that's your problem. :)

**Comment by ryby6969** [comment]
Thank you, I cannot believe I missed that. For some reason I am still getting the error that it cannot find the "kitty in jeans jpg"

**Comment by mmarchetti2** [comment]
Oh, that might be because in line 17, you just have Image.open(filename). This would just cause it to look for the files, and not through a specific folder, aka 'originals'. Try changing it to 
im = Image.open(os.path.join('originals', filename)) 
(The os.path.join is because the test is being run on a Linux, not Windows.) That seems to fix it, but the check still errors out on mine too. Everyone seems to be having the same problem. :/

**Comment by ryby6969** [comment]
Thank you very much. I thought there was somewhere that was missing the filename but could not figure it out.

File: resizeAndAddLogo.py [hunk 3]
```diff
@@ -7,8 +7,35 @@
...
+    width, height = im.size
+
+#TODO: Check if file image needs to be resized.
+    if width > SQUARE_FIT_SIZE and height > SQUARE_FIT_SIZE:
 #TODO: Calculate the new width and height to resize to.
+        if width > height:
+            height = int((SQUARE_FIT_SIZE / width) * height)
+            width = SQUARE_FIT_SIZE
+
+    else:
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @ryby6969 shouldn't this `else` block line up with the previous `if`?

**Comment by ryby6969** [comment]
Wow, I looked at it for like 30 minutes and completely missed that. Thanks you very much.

**Comment by halfaro1** [comment]
@ryby6969 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job!

**Pull request closed without merging**